src/components/SecondaryPanes/CommandBar.css

```diff
@@ -4,6 +4,10 @@
   display: flex;
   height: 30px;
   overflow: hidden;
+  position: sticky;
+  top: 0;
+  z-index: 1;
+  background-color: white;
```

Comment:
We should use some variable (`--theme-body-background`?) to make this work with all themes.